backend/src/main/java/co/inventorsoft/scripty/model/dto/ResetPasswordDto.java

```diff
@@ -0,0 +1,25 @@
+package co.inventorsoft.scripty.model.dto;
+import co.inventorsoft.scripty.validation.PasswordMatches;
+import co.inventorsoft.scripty.validation.ValidPassword;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+import javax.validation.constraints.Email;
+
+/**
+ *
+ * @author Symyniuk
+ *
+ */
+
+@Getter
+@Setter
+@PasswordMatches
```

Comment:
Why this annotation is doubled here? I think it should be enough to put it on the filed